src/plugins/intel_npu/src/plugin/npuw/partitioning/online/snapshot.cpp

```diff
@@ -580,12 +901,35 @@ void Snapshot::cleanUpUniques() {
         completeRepeating(reptag_to_gset.first, reptag_to_gset.second);
     }
 
+    afterUniques();
+
+    LOG_INFO("Number of groups after compiler pass: " << graphSize());
+    LOG_INFO("DONE");
+}
+
+void Snapshot::afterUniques() {
+    LOG_INFO("Online partitioning: executing afterUniques pass...");
+    LOG_BLOCK();
```

Comment:
Technically it is not a pass if it is not part of the pass pipeline. And here you call it from `cleanUpUniques` directly.
Probably I do the same, IDK

So so far let's keep it as is

Reply:
Ok